src/VisualStudio/Core/Impl/Options/Style/NamingStyleOptionGrid.xaml

```diff
@@ -0,0 +1,212 @@
+﻿<local:AbstractOptionPageControl
+    x:Class="Microsoft.VisualStudio.LanguageServices.Implementation.Options.Style.NamingStyleOptionGrid"
+    x:ClassModifier="internal"
+    xmlns:local="clr-namespace:Microsoft.VisualStudio.LanguageServices.Implementation.Options"
+    xmlns:converters="clr-namespace:Microsoft.VisualStudio.LanguageServices.Implementation.Options.Converters"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:options="clr-namespace:Microsoft.VisualStudio.LanguageServices.Implementation.Options"
+    xmlns:imaging="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.Imaging"
+    xmlns:vs="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.14.0"
+    xmlns:imagingPlatformUI="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Imaging"
+    xmlns:imagecatalog="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.ImageCatalog"
+    mc:Ignorable="d" d:DesignHeight="279" d:DesignWidth="514">
+
+    <Grid>
```

Comment:
Somewhat concerned about loc/glob/accessibility/etc.  Would like to have someone go over this with a fine toothed comb.  (Maybe @dpoeschl when he gets back).